File: src/main/java/org/nette/neon/parser/NeonElementType.java
package org.nette.neon.parser;

import com.intellij.psi.tree.IElementType;
import org.nette.neon.NeonLanguage;
import org.jetbrains.annotations.NotNull;


public class NeonElementType extends IElementType {
	public NeonElementType(@NotNull String debugName) {
		super(debugName, NeonLanguage.INSTANCE);
	}

	public String toString() {
		return "[Neon] " + super.toString();
	}
}

File: src/main/kotlin/org/nette/neon/parser/NeonElementType.kt
package org.nette.neon.parser

import com.intellij.psi.tree.IElementType
import org.nette.neon.NeonLanguage

class NeonElementType(debugName: String) : IElementType(debugName, NeonLanguage.INSTANCE) {
    override fun toString(): String {
        return "[Neon] " + super.toString()
    }
}
